src/PAModel/Serializers/SourceSerializer.cs

```diff
@@ -38,8 +38,9 @@ internal static partial class SourceSerializer
         // 16 - Group Control transform
         // 17 - Moved PublishOrderIndex entirely to Entropy 
         // 18 - AppChecker result is not part of entropy (See change 0.5 in this list) 
-        // 19 - Switch extension to .fx.yaml 
-        public static Version CurrentSourceVersion = new Version(0, 19);
+        // 19 - Switch extension to .fx.yaml  
+        // 20 - Only load themes that match the specified theme name 
+        public static Version CurrentSourceVersion = new Version(0, 20);
```

Comment:
Is this actually a breaking change? It looks like just a change to the Load path, but not the save path.

Reply:
It is, yeah. An app written to source in v19 that gets loaded in v20, if it was using a theme other than the default, won't have the property values that were considered default re-added in the same way.